docs/ccoctl.md

```diff
@@ -335,4 +335,89 @@ For alibaba cloud,  the CCO utility (`ccoctl`) binary will create credentials Se
    ```
    where:
    - `name` is the name used to tag any cloud resources that are created for tracking. 
-   - `region` is the Alibaba Cloud region in which cloud resources will be created.   
\ No newline at end of file
+   - `region` is the Alibaba Cloud region in which cloud resources will be created.   
+
+## Nutanix
+
+This is a guide for using manual mode on Nutanix, for more info about manual mode, please refer to [cco-mode-manual](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).
+
+For Nutanix,  the CCO utility (`ccoctl`) binary will create credentials Secret manifests for the OpenShift installer.
+
+### Procedure
+
+1. Extract the list of CredentialsRequest custom resources (CRs) from the OpenShift Container Platform release image:
+
+   ```bash
+   $ oc adm release extract --credentials-requests --cloud=nutanix --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+   ```
+
+   >  step 2&3 are only needed when preparing for upgrading clusters with manually maintained credentials. When doing a fresh installation please skip step 2&3**
+
+2. For each CredentialsRequest CR in the release image, ensure that a namespace that matches the text in the spec.secretRef.namespace field exists in the cluster. This field is where the generated secrets that hold the credentials configuration are stored.
+
+   Sample Nutanix CredentialsRequest object
+   ```yaml
+   apiVersion: cloudcredential.openshift.io/v1
+   kind: CredentialsRequest
+   metadata:
+     annotations:
+       include.release.openshift.io/self-managed-high-availability: "true"
+     labels:
+       controller-tools.k8s.io: "1.0"
+     name: openshift-machine-api-nutanix
+     namespace: openshift-cloud-credential-operator
+   spec:
+     providerSpec:
+       apiVersion: cloudcredential.openshift.io/v1
+       kind: NutanixProviderSpec
+     secretRef:
+       name: nutanix-credentials
+       namespace: openshift-machine-api
+   ```
+
+3. For any `CredentialsRequest` CR for which the cluster does not already have a namespace with the name specified in `spec.secretRef.namespace`, create the namespace:
+
+   ```bash
+   $ oc create namespace <component_namespace>
+   ```
+
+4. Ensure the following environment variables are set:
```

Comment:
do these need to be set permanently or just while doing this procedure?

Reply:
Just during the procedure.